main.go

```diff
@@ -184,3 +188,19 @@ func handleForceClose() {
 
 	launchTUI()
 }
+
+func handleOutputAll(format string) {
+	items := config.TextItems()
+
+	if format == "raw" {
+		for _, v := range items {
+			fmt.Printf("%q\n", v.Value)
+		}
+	} else if format == "unescaped" {
+		for _, v := range items {
+			fmt.Println(v.Value)
+		}
+	} else {
+		fmt.Printf("Invalid argument to -output-all\nSee %s --help for usage", os.Args[0])
```

Comment:
utils.LogError is also used for other error reporting

Reply:
utils.LogError logs to a file, I don't think that's the thing you would want to do if it's an error concerning command line arguments